### PR TITLE
Notify users about evaluation code

### PR DESCRIPTION
**Pull request by yuyu2172**
Follows discussion in #297.

### CODE REVIEW

File: examples/detection/README.md [hunk 1]
```diff
@@ -31,3 +31,11 @@ You can generate these visualization results by the following command.
 ```
 $ python visualuze_models.py
 ```
+
+## Notes on writing your own evaluation code
+
+Here are list of important configurations to reproduce results.
+
++ `model.use_preset('evaluate')` changes the score thresholds for NMS.
```
Comment (Hakuyume) [inline examples/detection/README.md]: `score_thresh` is not used in NMS. How about `configure postprocessing parameters for evaluation. (e.g. threshold of confidence score)`?

File: examples/detection/README.md [hunk 2]
```diff
@@ -31,3 +31,11 @@ You can generate these visualization results by the following command.
...
++ `DetectionVOCEvaluator` should be instantiated with `use_07_metric=True` (default is False), if evaluation is conducted on VOC07 test dataset.
```
Comment (Hakuyume) [inline examples/detection/README.md]: `VOC07` -> `VOC 2007`

File: examples/detection/README.md [hunk 3]
```diff
@@ -31,3 +31,11 @@ You can generate these visualization results by the following command.
...
++ When evaluating on VOC dataset, `VOCDetectionDataset` should return information about image difficulty, as the evaluation metric expects that to be included. The dataset returns it by setting `use_difficult=True` and `return_difficult=True`.
```
Comment (Hakuyume) [inline examples/detection/README.md]: `image difficulty` -> `difficulties of bounding boxes`? (difficulty is not for image)

Comment (zori) [inline examples/detection/README.md]: My bad, I've misunderstood the `difficult` in that case

File: examples/detection/README.md [hunk 4]
```diff
@@ -31,3 +31,12 @@ You can generate these visualization results by the following command.
 ```
 $ python visualuze_models.py
 ```
+
+## Notes on writing your own evaluation code
+
+Here are list of important configurations to reproduce results.
```
Comment (Hakuyume) [inline examples/detection/README.md]: `Here is a list`?

**Comment by yuyu2172** [comment]
Fixed

**Pull request merged**